build.js

```diff
@@ -0,0 +1,57 @@
+// Dependencies
+var GitHub = require("gh.js")
+  , SameTime = require("same-time")
+  , Logger = require("bug-killer")
+  , Fs = require("fs")
+  ;
+
+// A token is almost mandatory
+if (!process.argv[2]) {
+    Logger.log("Usage: node build.js <token>", "warn");
+    Logger.log("If a token is not provided, this script will fail due to the API rate limit.", "warn");
+}
+
+// Initialize the gh.js instance
+var gh = new GitHub({ token: process.argv[2] });
+
+// Fetch GitHub members
+Logger.log("Fetching the GitHub members.");
+gh.get("orgs/github/members", { all: true }, function (err, data) {
```

Comment:
> Sadly I know nothing about how to use the GitHub API.

Ha! GitHub API is probably the best API I have ever used. Everything is so natural. `{ all: true}` tells [`gh.js`](https://github.com/IonicaBizau/gh.js) to handle pagination automatically (max is 100 per page), so more requests are done in the background. :sparkle: